src/current/v24.3/set-up-logical-data-replication.md

```diff
@@ -0,0 +1,213 @@
+
+{{site.data.alerts.callout_info}}
+If you would like to ignore TTL deletes in LDR, you can use the `discard = ttl-deletes` option in the `CREATE LOGICAL REPLICATION STREAM` statement.
+{{site.data.alerts.end}}
+
+1. From the **destination** cluster, start LDR. Use the fully qualified table name and ensure the source and destination table names are identical:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    CREATE LOGICAL REPLICATION STREAM FROM TABLE {database.public.table_name} ON 'external://{source_external_connection}' INTO TABLE {database.public.table_name};
+    ~~~
+
+    You can change the default `mode` using the `WITH mode = validated` syntax.
+
+    If you would like to add multiple tables to the LDR job:
+
+        {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    CREATE LOGICAL REPLICATION STREAM FROM TABLES ({database.public.table_name},{database.public.table_name},...)  ON 'external://{source_external_connection}' INTO TABLES ({database.public.table_name},{database.public.table_name},...);
+    ~~~
+
+    {% include {{ page.version.version }}/ldr/multiple-tables.md %}
+
+    Once LDR has started, an LDR job will start on the destination cluster. You can [pause]({% link {{ page.version.version }}/pause-job.md %}), [resume]({% link {{ page.version.version }}/resume-job.md %}), or [cancel]({% link {{ page.version.version }}/cancel-job.md %}) the LDR job with the job ID. Use `SHOW LOGICAL REPLICATION JOBS` to display the LDR job IDs:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    SHOW LOGICAL REPLICATION JOBS;
+    ~~~
+    ~~~
+            job_id        | status  |          targets          | replicated_time
+    ----------------------+---------+---------------------------+------------------
+    1012877040439033857   | running | {database.public.table}   | NULL
+    (1 row)
+    ~~~
+
+    If you're setting up bidirectional LDR, both clusters will have a history retention job and an LDR job running.
+
+1. Move on to [Step 4](#step-4-optional-set-up-bidirectional-ldr) to set up a second LDR job. Or, once you have set up your required LDR jobs, refer to [Step 5](#step-5-monitor-the-ldr-jobs) to monitor the jobs in the DB Console.
+
+## Step 4. (Optional) Set up bidirectional LDR
+
+At this point, you've set up one LDR job from cluster A as the source to cluster B as the destination. To set up LDR streaming in the opposite direction, complete [Step 2](#step-2-connect-from-the-destination-to-the-source) and [Step 3](#step-3-start-ldr) again. Cluster B will now be the source, and cluster A will be the destination.
+
+    {% comment  %} TODO: Add output here when example included.
+    To verify that data at a certain point in time is correct on the destination cluster, refer to [Data verification](). 1. After the two jobs are running, you can check the number of rows on each cluster with  {% endcomment %}
+
+## Step 5. Monitor the LDR jobs
```

Comment:
Should we also call out that metrics export is possible?

Reply:
Added a tip with all the monitoring tools here because the monitoring page won't be published quite a as this page goes out. I will change this when the monitoring page is published soon after.